camel-route-support/src/main/java/io/kaoto/backend/api/service/step/parser/camelroute/IntegrationStepParserService.java

```diff
@@ -81,6 +80,16 @@ public ParseResult<Step> deepParse(final String input) {
         return res;
     }
 
+    @Override
+    public List<ParseResult<Step>> getParsedFlows(String input) {
+        var res = new LinkedList<ParseResult<Step>>();
+        String[] splitCRDs = input.split(System.lineSeparator() + "---" + System.lineSeparator());
```

Comment:
Maybe this is too weak? Not sure how to make it better, to be honest. This could be in the middle of a long string text. Evil people will do evil things, you know.

Reply:
yes, haters gonna hate, I know... Would something [like this](https://github.com/FasterXML/jackson-dataformats-text/issues/66#issuecomment-1386673293) help?

Reply:
Let me try :)

Reply:
Not really working as expected.